book/from_flat_php_to_symfony2.rst

```diff
@@ -399,13 +422,11 @@ routing, calling controllers, templates, etc.). More time will need to be
 spent to handle form submissions, input validation, logging and security.
 Why should you have to reinvent solutions to all these routine problems?
 
-.. _add-a-touch-of-symfony2:
```

Comment:
this should not be removed, as it would break existing anchors (this is precisely why it was added)